zkevm-circuits/src/evm_circuit/execution/returndatacopy.rs

```diff
@@ -187,8 +188,8 @@ impl<F: Field> ExecutionGadget<F> for ReturnDataCopyGadget<F> {
             (4, CallContextFieldTag::LastCalleeReturnDataOffset),
             (5, CallContextFieldTag::LastCalleeReturnDataLength),
         ]
-        .map(|(i, tag)| {
-            let rw = block.get_rws(step, i);
+        .map(|(_, tag)| {
+            let rw = rws.next();
             assert_eq!(rw.field_tag(), Some(tag as u64));
             rw.call_context_value()
         });
```

Comment:
This whole block from line 185 can be replaced with:
```
let last_callee_id = rws.next().call_context_value();
let return_data_offset = rws.next().call_context_value();
let return_data_size = rws.next().call_context_value();
```

Reply:
Thanks! I missed this